app/index.html

```diff
@@ -24,6 +24,8 @@
   <script type="text/javascript" src="js/livebus/livebus.factory.js"></script>
 
   <script type="text/javascript" src="js/shared/shared.module.js"></script>
+  <script type="text/javascript" src="js/shared/ucBackground/ucBackground.module.js"></script>
+  <script type="text/javascript" src="js/shared/ucBackground/ucBackground.directive.js"></script>
```

Comment:
I just thought that the word image would make it clear right from the filename that it is manipulating the background image. But I don't mind if you remove it.

> "All other conventions still hold using a suffix of the type" and I can't find a mention of other suffixes.

That part seems to say that you can drop the `controller` postfix but not with other types (such as services, providers, factories, components, directives). I think we agree on this? :)

> `background.directive.js` or `background-image.directive.js`

Both are fine with me.

Just dropping the `uc` from everywhere else except the directive name and then everything fits with the rest of the project naming.